clangd-tidy-ci.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+# Clangd by default will not run "expensive" checks. So we need two config
+# files: one for the language server; the other for the CI.
+# Because clangd-tidy does not support separate paths for config files,
+# we break these into separate files and do this moving hack.
+
+cp .clangd .clangd-orig
+echo "Diagnostics:" >> .clangd
+echo "  ClangTidy:" >> .clangd 
+echo "    FastCheckFilter: None" >> .clangd
```

Comment:
Why not include this in .clangd all the time. If we do experience a slowdown in the IDE LSP as a result, we can remove this line later.

Reply:
OK. A little premature optimization on my part.